documents/update_ssl_certificates.md

```diff
@@ -0,0 +1,25 @@
+# Updating the SSL Certificates on Overcloud
+
+### undercloud
```

Comment:
Ah, so the certificates are generated on the undercloud! I'd rename this to be clearer "Generate certificates on the undercloud" here and "Deploy certificates on the overcloud" below.

Reply:
Done.